packages/client/src/options.ts

```diff
@@ -7,20 +7,27 @@ import {QParamType} from './QueryApi'
 export interface ConnectionOptions {
   /** base host URL */
   host: string
+
   /** authentication token */
   token?: string
+
   /** token authentication scheme. Not set for Cloud access, set to 'Bearer' for Edge. */
   authScheme?: string
+
   /**
    * socket timeout. 10000 milliseconds by default in node.js. Not applicable in browser (option is ignored).
    * @defaultValue 10000
+   * @Deprecated: Please use more specific properties like WriteOptions.timeout.
    */
   timeout?: number
```

Comment:
In addition to deprecating this field, perhaps a more specific field `writeTimeout` can also be added.  The value of `timeout` would just be assigned to the newer `writeTimeout`.  `WriteOptions.timeout` can also be used, and since it is more specific, could take precedent.